client/css/style.css

```diff
@@ -1475,6 +1475,20 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	display: inline-block;
 }
 
+.whois {
+	display: grid;
```

Comment:
Look at us, jumping on the grid layout train!
That kills Edge 15 support though, but :man_shrugging: Edge 16 was released in October, so that should be fine.

Reply:
> Edge 16 was released in October, so that should be fine.

Awfully optimistic about that as it is tied in with the fall creators update people have to install. Not to mention people limited by the IT department at their work. 
I am not saying the lounge should support older browser until the end of times, but I'd say this might be a bit soon as far as browser support goes. 

Seems like you can be a bit more in the clear using a -ms- prefix here.

Reply:
Yeah, `-ms-` for grid layout is reasonable. We usually support "last 2 browsers", though we have no test running on browsers to kind of prove it (nor correct tests for that) yet.